src/client/components/ReviewPost/ReviewPost.styles.css

```diff
@@ -0,0 +1,35 @@
+.review-section {
+  max-width: 600px;
+  display: flex;
+  padding: 1.5em;
+}
+.profile-image {
+  width: 140px;
+  height: 100px;
```

Comment:
Just choose one: either width or height, don't change both because then the image gets squished 
```suggestion
```